maven-enforcer-plugin/src/main/java/org/apache/maven/plugins/enforcer/EnforceMojo.java

```diff
@@ -127,11 +127,22 @@ public class EnforceMojo extends AbstractMojo {
     private PlexusConfiguration rules;
 
     /**
-     * List of strings that matches the EnforcerRules to execute.
+     * List of strings that matches the EnforcerRules to execute. Use <code>enforcer.rules</code> property instead.
+     *
+     * @deprecated
```

Comment:
should be ` @deprecated descriptions ....`

please check `mvn site` output how documentation will be generated

Reply:
Yeah, I had that previously, but the description is then not present in main table with the goal parameters, it only gets added to the "Parameter Details" list (which I missed before). I'll add here as well.

I did check the generated documentation. Looks good to me, after the suggested fix.